designer/server/src/common/templates/partials/navigation.njk

```diff
@@ -4,6 +4,7 @@
 {{ appServiceHeader({
   organisationName: "Defra",
   productName: "Forms Designer",
+  env: env,
```

Comment:
Might as well pass `config` here so it's easier to search for `{{ config.env }}`

Or if you're happy that `config` is already in the global context, you don't even need a param

```suggestion
  config: config,
```

Reply:
config is not in global context. I am passing just env for now. If we need more from config it can be changed in future.